style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.container {
+  background: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.part1 {
+  border: 1px solid rgb(202, 199, 199);
+  background: url(images/background.jpeg);
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  margin-bottom: 5%;
+  align-items: center;
+}
+
+.head-name {
+  font-family: 'Poppins', sans-serif;
+  font-size: 48px;
+  font-weight: 700;
+  margin: 40% 10px 17px;
+  line-height: 50px;
+  right: 10px;
+  display: flex;
+}
+
+.head-para {
+  font-family: 'Poppins', sans-serif;
+  left: 20px;
+  right: 10px;
+  margin: 10px 17px;
+}
```

Comment:
- [ ] Please stick to the design as much as possible 💯 both the headline title and the paragraph color and size should be identical to the Figma design 💪🏼 

### Your Design:
![Screenshot from 2023-03-21 23-46-23](https://user-images.githubusercontent.com/68030297/226758693-de84db49-0d62-4466-90b5-78bf3707d28e.png)

### Figma Design:
![Screenshot from 2023-03-21 23-47-31](https://user-images.githubusercontent.com/68030297/226758857-2a85c5db-663a-4a25-a985-3ae3b2f357c0.png)